packages/fxa-graphql-api/pm2.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       min_uptime: '2m',
       env: {
         NODE_ENV: 'development',
-        NODE_OPTIONS: '--inspect=9190',
+        NODE_OPTIONS: '--inspect=9290',
```

Comment:
This was conflicting with fxa-admin-server.

Reply:
Note, the README in the fxa/ root, not in this package.